src/test_l3_dsAudio.c

```diff
@@ -1798,11 +1857,11 @@ void test_l3_dsAudio_ms12Profile(void)
     int32_t choice = -1;
     intptr_t handle = (intptr_t)NULL;
     bool enabled = false;
-    dsMS12AudioProfileList_t profiles = {0};
-    char *profileList[DS_AUDIO_MAX_MS12_PROFILES] = {NULL};
-    char profile[DS_AUDIO_MAX_MS12_LENGTH]  = {0};
+    char *profileList[] = {"Off", "Music" ,"Movie", "Sports" , "Entertainment", "Night", "Party" , "User"};
```

Comment:
Where does this list come from? And how will it change in the future? static lists should really be const.